tools/test_runner/selected_tests_to_test_spec.erl

```diff
@@ -22,11 +22,19 @@ main(AtomArgs) ->
 small_specs(AtomArgs) ->
     lists:filter(fun is_small_spec/1, AtomArgs).
 
+big_specs(AtomArgs) ->
+    lists:filter(fun is_big_spec/1, AtomArgs).
+
 is_small_spec(Atom) ->
     FileName = atom_to_list(spec_to_module(Atom)) ++ ".erl",
     Path = filename:join("test", FileName),
     does_file_exist(Path).
 
+is_big_spec(Atom) ->
+    FileName = atom_to_list(spec_to_module(Atom)) ++ ".erl",
+    Path = filename:join("big_tests/tests", FileName),
+    does_file_exist(Path).
```

Comment:
There is no agreement, that SUITE-s are located in "big_tests/tests" only.
Some people store client specific tests in "big_tests/feature" folder.

The fully correct way would be to consult with spec file.

But I guess it's fine too.